platform.md

```diff
@@ -915,6 +996,7 @@ id = "<buildpack ID>"
 version = "<buildpack version>"
 api = "<buildpack API version>"
 homepage = "<buildpack homepage>"
+extension = false
```

Comment:
_Take this with a grain of salt since I may not fully understand the purpose of this file._

This feels like we are conflating extensions with buildpacks. Should there be a set of entries specific for extensions? ie. `[[extensions-group]]`.

Reply:
That is certainly an option. Or something like `[[group-extensions]]` for parity with the suggested ``[[order-extensions]]`?